.github/workflows/nightly-docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Nightly Documentation Build
+
+on:
+  schedule:  # UTC at 0400
+    - cron:  '0 4 * * *'
+
+jobs:
+  docs_build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.8
+
+      - name: Install pyaedt
+        run: |
+          pip install .
+
+      - name: Install doc build requirements
+        run: |
+          pip install -r requirements_docs.txt
+
+      - name: Full Documentation Build
+        run: |
+          make -C doc phtml
```

Comment:
@akaszynski, is it supposed to be:
`make -C doc phtml`
or
`make -C doc html`
?

Reply:
See the makefile.  Effectively, `phtml` is running a parallel build of sphinx using as many cores available.  Github agents run https://docs.microsoft.com/azure/virtual-machines/dv2-dsv2-series#dsv2-series, meaning we get once genuine core and one hyperthreaded.  There's some small improvement, so its better than nothing.